src/gwt/src/org/rstudio/core/client/VirtualConsole.java

```diff
@@ -39,17 +40,38 @@
  */
 public class VirtualConsole
 {
+   // don't do any processing of ANSI escape codes
+   public final static int ANSI_COLOR_OFF = 0;
+   
+   // convert ANSI color escape codes into css styles
+   public final static int ANSI_COLOR_ON = 1;
+   
+   // strip out ANSI escape sequences but don't apply styles
+   public final static int ANSI_COLOR_STRIP = 2;
+  
    public VirtualConsole()
    {
-      this(null);
+      this(null, ANSI_COLOR_ON);
    }
    
    public VirtualConsole(Element parent)
    {
+      this(parent, ANSI_COLOR_ON);
+   }
+
+   /**
+    * VirtualConsole constructor
+    * @param parent parent element
+    * @param showAnsiColors if true, apply visual styles for AnsiColor escape
```

Comment:
nit: documentation seems to imply this is a boolean, but is really an `enum`-flavored `int`